components/events/EventCard.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <div class="card">
+        <div class="title_block">
+            <div class="title">{{ title }}</div>
+            <div class="tag" :class="[tagColor]">{{ tag }}</div>
```

Comment:
Consider using a computed property to get the tag's class for scalability.

In script, e.g.,

```js
computed: {
  tagClasses() {
    return [this.tagColor]
  },
}
```

See also: https://vuejs.org/v2/guide/class-and-style.html

Reply:
fixed